runtime/src/accounts_db.rs

```diff
@@ -2955,53 +2910,14 @@ impl AccountsDb {
         hasher.result()
     }
 
-    pub fn hash_account_data(
+    fn hash_account_data(
```

Comment:
It's hard to tell from the Github diff; I've basically just renamed the `blake3_hash_account_data()` function to `hash_account_data`. (And also removed `pub`, because it doesn't look like anything else was/is calling this function.)